ui/apps/platform/src/Containers/Dashboard/PatternFly/Widgets/ComplianceLevelsByStandard.test.tsx

```diff
@@ -108,36 +95,45 @@ describe('Compliance levels by standard dashboard widget', () => {
         // Allow graph to load
         await screen.findByLabelText('Compliance coverage by standard');
 
-        async function getBarPercentages() {
-            // eslint-disable-next-line testing-library/no-node-access
-            const svgBarsElement = document.querySelector('svg > g:nth-of-type(3)');
-            const barPercentages = await within(svgBarsElement as HTMLElement).findAllByText(
-                /\d+%/
-            );
-            expect(barPercentages).toHaveLength(6);
+        async function getBarTitles() {
+            const titlesRegex = new RegExp(`${Object.values(standardNames).join('|')}`);
```

Comment:
If I understand correctly, find all possible standard names here, and verify below only the expected subset?

Is there an HTML element or accessibility attribute to search for titles without a pre-expectation of their content?

Reply:
Unfortunately there isn't an accessibility attribute to grab onto, or another RTL utility that can be used to query these elements, since the HTML is SVG code generated by the Victory library:

Roughly
```
<g role="presentation">
  <line vector-effect="non-scaling-stroke" role="presentation" shape-rendering="auto" style="..."></line>
  <a href="...">
    <text direction="inherit" dx="0" x="135" y="214.97" id="chart-axis-0-tickLabels-0">
      <tspan x="135" dx="0" dy="0" text-anchor="end" style="...">CIS Kubernetes v1.5</tspan>
    </text>
  </a>
</g>
```

We could grab onto the `role="presentation"` attribute, but that would also pick up the other elements of the chart and we would still need to filter out the percentages, etc.

We could query directly by DOM structure, but I figured that would be more subject to breakages outside of our control than using the name constants defined earlier in the test file.

Reply:
Yeah, better to leave well enough alone. A lesson for the future when you contribute to an open source library!